### ISSUE TITLE
CodeDay workshop attendance

### ISSUE THREAD

**Issue by mikelbrierly** [developer-delta/halogen#82]
## User Story:

> As a developer,
I want to attend weekly workshops,
So that I can learn and help to advance my career

---

## Acceptance Criteria:

- [x] Attend one CodeDay talk or workshop
  - Add a comment to this story mentioning the name of the workshop, and something interesting you learned 
- [x] Attend a second CodeDay talk or workshop
  - Add a comment to this story mentioning the name of the workshop, and something interesting you learned

**Comment by Hanna-N9** [comment]
I watched a video called "Mistakes I made in tech so you don’t have to" from CodeDay Youtube that came up around 2 weeks ago. This was the Talk I appreciated. I didn't know there is what is called imposter syndrome. I learned the definition of it is, doubting yourself, your abilities, feeling overall like you're a fraud which was interesting to know. Another good to know is to put the project that you can feel confidence and comfortable to talk about and know about. Don’t put the project that you felt you had a minimal role because it can bites you during the interview. Also talk about the projects and previous work experiences you have with recruiters. Not about how much time you spend on lead code, working on code challenges. Those were the main ones that are good to know and to keep in mind.

**Comment by mikelbrierly** [comment]
I'm glad you got to watch the recording for this one! It sound like it was really helpful. Imposter syndrome is something everyone faces, and the more you learn that everyone has had to deal with it, the easier it is to overcome.

**Comment by Hanna-N9** [comment]
I watched a video called "Culture Fit and Interviewing Soft Skills," which taught me a lot about culture fit and what I should do during the interviews. I learned that culture fit is a norm in companies and that they keep that way on purpose, which I found interesting. Furthermore, I learned that it is natural and normal to be nervous or uncomfortable during the interview, and because of that, most interviewers know that they would expect that. They're not going to take an interviewee being nervous as a sign that there's something wrong or inadequate with that person. Because they've been there, and they've all been through that. What matters is how you present to the interviewer and how you respond to that anxiety. Before the interview, it's a good practice thinking of yourself as somebody who fundamentally is supposed to be there and is good enough to be there because that's the key.

If you don't know or aren't familiar with questions, don't disengage, redirecting to something you know well. Also, don't try to mislead or act like you know what you're talking about when you aren't sure. The speaker gave us advice that was helpful on how to approach each of them that I took notes of. For example, don't respond defensively and don't say, "I don't know how to do that," because that will be how the interviewer sees you and get the sense that will imply that you're not willing to try to learn new things and you might be closing yourself that way. Instead, be curious, identify relevant skills, and talk through finding answers. The way an interviewee responds to them more even than the content of the question tells an interviewer a lot of information about them that the interviewer really wants to know.
